Description: Adapt libasyncns.pc(.in) for multiarch support
    In multiarch packages, the libdir is changed to /usr/lib/<triplet>. This
    pkg-config must thus use the @libdir@ variable directly to get it
    correctly.
Author: Tanguy Ortolo <tanguy+debian@ortolo.eu>
Forwarded: not-needed
Last-Update: 2011-07-30

--- libasyncns-0.8.orig/libasyncns.pc.in
+++ libasyncns-0.8/libasyncns.pc.in
@@ -1,7 +1,7 @@
 prefix=@prefix@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
-includedir=${prefix}/include
+libdir=@libdir@
+includedir=@includedir@
 
 Name: libasyncns
 Description: Asynchronous Name Service
